src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  padding: 32px 16px 16px;
+  background: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    align-self: center;
+  }
+
+  &__name {
+    margin-top: 40px;
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-dark-blue;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $color-gray;
+  }
+
+  &__stars {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: flex-end;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    text-align: right;
+    color: $color-dark-blue;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    display: flex;
+    align-items: flex-end;
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $color-gray;
+  }
+
+  &__sum {
+    display: flex;
+    align-items: flex-end;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    text-align: right;
+    color: $color-dark-blue;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    text-decoration: none;
+    text-transform: uppercase;
+    background: $color-blue;
+    color: $color-white;
+    border-radius: 5px;
+
+    &:hover {
+    color: $color-blue;
+    background: $color-white;
+    border: 1px solid $color-blue;
+    }
+  }
+
+}.card:not(:nth-child(4n)) {
+  margin-right: $margin-right;
+}
+
+.card:not(:nth-last-child(-n + 4)) {
+  margin-bottom: 46px;
+}
```

Comment:
do not apply margins or any external geometry directly to the BEM block use BEM mixes instead. Fix it everywhere